lib/child-writer/index.js

```diff
@@ -194,6 +244,7 @@ exports.write = (image, drive, options) => {
 
     child.on('close', (code) => {
```

Comment:
Just out of curiosity, is this event triggered when we kill the child?

Reply:
It was before `removeAllListeners` gets called, but not anymore, good catch! It exits cleanly now.